README.md

```diff
@@ -3,7 +3,7 @@
 > neotype - a type specimen that is selected subsequent to the description of a species to replace a preexisting type that has been lost or destroyed.
 
 ![CI](https://github.com/neotypes/neotypes/workflows/CI/badge.svg?branch=main)
-[![Maven Central](https://img.shields.io/maven-central/v/com.dimafeng/neotypes_2.12.svg)](https://maven-badges.herokuapp.com/maven-central/com.dimafeng/neotypes_2.12)
+[![Maven Central](https://img.shields.io/maven-central/v/io.github.neotypes/neotypes-core_2.12.svg)](https://maven-badges.herokuapp.com/maven-central/io.github.neotypes/neotypes-core_2.12)
```

Comment:
I am not 100% sure if this will be the new URL of the badge.

Reply:
I'm afraid that the only way to check it is to release the version 😁